assignment/scripts/aboutMe.js

```diff
@@ -1,44 +1,61 @@
 // REQUIRED FEATURES:
 // 1 - Create a variable called `firstName` and assign it the value of your first name
-
+let firstName= 'Brandon'
 // 2 - Create a second variable called `lastName` and assign it the value of your last name
-
+let lastName= 'Paulson'
 // 3 - Create a third variable called `fullName`, assign it the value of your first and last name
 // (remember, you already have variables for this, can you use those?)
-
+let fullName= (firstName+lastName)
 // 4 - Console log the value of `fullName`
-
+console.log(fullName)
 // 5 - Create a variable called `luckyNumber` and assign it the value of your lucky number.
-
+let luckyNumber=7
 // 6 - Console log this sentence, adding in the variables you created above: 
 // 'My name is (full name), and I think (lucky number) is a winner!'.
 // Refer back to the videos if you need help with this one.
-
+console.log('My name is '+fullName+ ', and I think '+luckyNumber+ ' is a winner!')
```

Comment:
Again, consistent spacing can go a long way towards making your code more readable for future devs.

```js
console.log('My name is ' + fullName + ', and I think ' + luckyNumber + ' is a winner!');
```